peachjam/templates/peachjam/_registries.html

```diff
@@ -0,0 +1,19 @@
+{% load i18n %}
+{% if registries %}
+  <h4 class="my-4">Registries</h4>
+  {% for group in registry_groups %}
```

Comment:
a `.col-*` must always be included in a `.row` -- add `<div class="row">` above the for loop. that will also create the columns that you're missing.